packages/showcase/src/app/components/tooltip/metadata.ts

```diff
@@ -0,0 +1,95 @@
+import { ComponentMetadata } from "../../../common/types";
+import { chameleonImportType } from "../../../services/template-language/create-template";
+
+import { CSS } from "../../../common/html";
+
+export const tooltipMetadata = {
+  title: "Tooltip (work in progress)",
+  description:
+    "A tooltip in web development is a small, interactive pop-up that appears when a user hovers over or focuses on an element. It provides additional information or context without cluttering the interface, offering users quick insights about features or actions. Tooltips are commonly used to enhance usability by clarifying unclear elements or providing guidance.",
+  codeSnippets: {
+    blockStart: {
```

Comment:
There are 3 ways of using the `ch-tooltip` in Chameleon and this way is the least recommended for me, because it's more difficult to customize the action button.

It think we should show use this way:
```tsx
<button class="button-secondary" type="button">
  Something 2
  <ch-tooltip
    actionElement={null}
    class="tooltip"
  >
    Tooltip inside a button element
  </ch-tooltip>
</button>
```